mx.sulong/suite.py

```diff
@@ -215,6 +215,14 @@
       ],
       "sha1" : "5552052e53fcd7fe40c558866c9cd51027c17322",
     },
+    "LLVMTESTSUITE" : {
+      "path" : "tests/test-suite-3.2.src.tar.gz",
+      "urls" : [
+        "https://lafo.ssw.uni-linz.ac.at/pub/sulong-deps/test-suite-3.2.src.tar.gz",
+        "http://llvm.org/releases/3.2/test-suite-3.2.src.tar.gz",
```

Comment:
We cannot have any `http` links due to security reasons. Can you remove the link from `llvm.org`? There is a `http-check` command but I think it does not check the `suite.py` file. Can you add the check for this file?